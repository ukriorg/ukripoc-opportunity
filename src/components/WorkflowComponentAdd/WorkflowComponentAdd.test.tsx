import React from "react";
import renderer from "react-test-renderer";
import { WorkflowComponentAdd } from "./index";

describe("WorkflowComponentAdd", () => {
    it("renders correctly", () => {
        const tree = renderer
            .create(<WorkflowComponentAdd opportunityId={"1"} />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
