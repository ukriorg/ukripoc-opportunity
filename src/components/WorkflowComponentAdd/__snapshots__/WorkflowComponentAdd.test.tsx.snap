// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WorkflowComponentAdd renders correctly 1`] = `
<div>
  <p>
     Add workflow component
  </p>
  <label
    className="sc-bdVaJa kJhFvF"
    name="group1"
  >
    <span
      className="sc-bwzfXH lbxcXc"
    />
    <select
      className="sc-ifAKCX kDzsBS"
    >
      <option
        value="0"
      >
        Website Listing
      </option>
    </select>
  </label>
  <button
    className="sc-EHOje hEaLol"
    disabled={false}
  >
     Save 
  </button>
</div>
`;
